### ISSUE TITLE
Any instructions about the CelebA experiments?

### ISSUE THREAD

**Issue by MingSun-Tse** [huawei-noah/Data-Efficient-Model-Compression#9]
Hi, Thanks for your very helpful codes. I have two issues. (1) Do you have any README instructions about how to do the CelebA experiments in the paper? (2) Could you release the trained teacher and student models? I trained myself the CelebA teacher model but found the test accuracy is surprisingly higher, up to ~90%, using the AlexNet-based architecture. I don't know if I do it right or actually sth wrong... Thank you very much!!

**Comment by MingSun-Tse** [comment]
For any possible help: Turns out that ~90% accuracy was because I classify ALL the 40 CelebA attributes, which are seriously unbalanced. Training the teacher on "the most balanced attribute" will give the right teacher accuracy of ~81%.

**Comment by MingSun-Tse** [comment]
Sorry to bother again. DAFL training of the student on CelebA does not work well. I tried two hyper-parameter settings:
(1) ie=5, a=0.1, oh=1, batch size=512, lr_G=lr_S=1e-4, Adam solver. Result: does not converge at all.
(2) ie=5, a=1, oh=0.1, batch size=512, lr_G=lr_S=1e-4, Adam solver. Result: converged. But accuracy is low, lingering at ~0.56-0.60 after 50 epochs.
How should I reproduce the results in the DAFL paper, any guidance? Thanks in advance!

**Comment by HantingChen** [comment]
I can not remember the exactly hyper-parameters for CelebA for now. You can try oh = 0.1~0.3 ie = 1 a = 0.1~0.01 batch size = 512 latent_dim=1000.
The teacher model and instruction for CelebA will be released after deadline of CVPR.
